examples/hello-world-pyprojecttoml/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["setuptools", "wheel", "setuptools-rust"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "hello-world"
+version = "1.0"
+
+[project.scripts]  # Python entry-point wrapper to be installed in `$venv/bin`
+sum-cli = "hello_world.sum_cli:main"
+
+[tool.setuptools.packages]  # pure Python packages/modules
+find = { where = ["python"] }
+
+[[tool.setuptools-rust.ext-modules]]
+target = "hello_world._lib"  # private module to be nested into Python package
+py-limited-api = "auto"
+args = ["--profile", "release-lto"]
+
+[[tool.setuptools-rust.binaries]]  # executable to be installed in `$venv/bin`
```

Comment:
I wonder if this should be `[[tool.setuptools-rust.bin]]` to match `Cargo.toml`'s `[[bin]]`?

Reply:
I don't have any strong feelings about this change.
Can we have an agreement on all the changes in terms so I can implement all of them in batch?

Reply:
I think `bin` is better than `binary` to match cargo, however I think given the comment below about plurals, let's go for `bins` perhaps? What do you think of that?

Reply:
No problems.

I implemented it in 1f58338d7134b93efe9bd9ff3e9dbd2e464dd726.
(`bins` might sound a bit weird, but I think that the argument you made about consistency is important).